deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -6,19 +6,27 @@
 
 
 def callback_detect_vulnerability_scan_started(line):
-    msg = r'.*wazuh-modulesd:vulnerability-detector:\s+INFO:\s+\(\d+\):\s+Starting\s+vulnerability\s+scanning'
+    msg = r'.*wazuh-modulesd:vulnerability-detector.* INFO:.*Starting vulnerability scanning'
     match = re.match(msg, line)
 
     return match is not None
 
 
 def callback_detect_vulnerability_scan_finished(line):
-    msg = r'.*wazuh-modulesd:vulnerability-detector:\s+INFO:\s+\(\d+\):\s+Vulnerability\s+scanning\s+finished'
+    msg = r'.*wazuh-modulesd:vulnerability-detector.* INFO:.*Vulnerability scanning finished'
     match = re.match(msg, line)
-
     return match is not None
 
 
+def callback_detect_vulnerability_scan_sleeping(line):
+    msg = r'.*wazuh-modulesd:vulnerability-detector.* Sleeping for (.*)...'
+    match = re.match(msg, line)
+
+    if match:
+        return match.group(1)
+    return None
```

Comment:
Is it necessary to return the matched string? Maybe you can just 
```suggestion
    return match is not None
```

Reply:
It is necessary. We've discussed it.

Reply:
Got it, but we can improve this a bit. 
```suggestion
    return match.group(1) if match is not None else None
```